packages/addons/README.md

```diff
@@ -0,0 +1,49 @@
+#Addon skeleton
+This is an example how the default structure of an addon should look like.
+Please be aware that this is an moving target and could be changed to adapt changes at the build system.
```

Comment:
Remove this line. If the structure changes this skeleton should be updated to reflect the changes.